MauiProgram.cs

```diff
@@ -1,20 +1,28 @@
-﻿using Microsoft.Extensions.Logging;
+﻿using DotNetMaui.Services;
+using Microsoft.Extensions.Logging;
 
 namespace DotNetMaui;
 
 public static class MauiProgram
 {
+	public static bool DidCrashOnLastRun { get; private set; }
+	
 	public static MauiApp CreateMauiApp()
 	{
 		var builder = MauiApp.CreateBuilder();
 		builder
 			.UseMauiApp<App>()
+			
 			// This adds Sentry to your Maui application
             .UseSentry(options =>
             {
                 // The DSN is the only required option.
-                options.Dsn = "https://eb18e953812b41c3aeb042e666fd3b5c@o447951.ingest.sentry.io/5428537";
+                options.Dsn = Configuration.SentryDsn;
                 options.Debug = true;
+                
+                // Block your user order data from going to Sentry
+                options.SendDefaultPii = false;

```

Comment:
We've been adding `true` on all code snippets in order to get IP addresses showing up in Sentry at least.
`false` is default, curious why we're going with that here? Could we go with `true` instead?

What we show u with `true` is set is not documented yet, but we're missing that in .NET.
related to:
* https://github.com/getsentry/sentry-docs/issues/10151

Reply:
Absolutely.  I had assumed you may want it set to false to hide order information in the setup.

Reply:
I think it's prob better to set it to `true`? We'd get the user IP address at least, right?